ttnn/cpp/ttnn/operations/reduction/generic/device/reduce_op.cpp

```diff
@@ -77,25 +79,34 @@ std::vector<tt::tt_metal::LegacyShape> Reduce::compute_output_shapes(const std::
             padding[3] = Padding::PadDimension{0, 31};
             break;
     }
-    return {tt::tt_metal::LegacyShape(output_shape, padding)};
+    auto output2 = tt::tt_metal::LegacyShape(output_shape, padding);
+    std::cout << output2 << std::endl;
```

Comment:
Can you please review this method and remove usage of LegacyShape inside?
Basically reduction makes some dim/s 1. This will look 100 times simpler.

Reply:
This is here for debug. It will only be `ttnn::SimpleShape` in this function and no padding :)